app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,22 @@
+class GuestsController < ApplicationController
+    def show
+      @guest = Guest.find(params[:id])
+    end
+  
+    def create
```

Comment:
This controller action looks great for building a new Guest, and it even includes error handling - nice work! However, it's not part of any user story. If you need to create guests in your database, you can try to create them in the seeds.rb file or in the Rails console to play around with

For US 2, we are asking for a `create` action but in that case we are creating the association between an existing guest and existing room, so that's creating a record in the joins table you create (`stays`) and would be better off living in a joins controller: GuestRoomsController